yaml/landuse.yaml

```diff
@@ -1,114 +1,482 @@
+  - filter: {landuse: recreation_ground}
+    min_zoom: *tier4_min_zoom
+    output: {kind: recreation_ground, tier: 4}
+  # retail
+  - filter: {landuse: retail}
+    min_zoom: *tier4_min_zoom
+    output: {kind: retail, tier: 4}
+  # stadium
+  - filter: {leisure: stadium}
+    min_zoom: *tier4_min_zoom
+    output: {kind: stadium, tier: 4}
+  # zoo
+  - filter: {tourism: zoo}
+    min_zoom: *tier4_min_zoom
+    output: {kind: zoo, tier: 4}
+  # wildlife_park
+  - filter: {zoo: wildlife_park}
+    min_zoom: *tier4_min_zoom
+    output: {kind: wildlife_park, tier: 4}
+  # winter_sports
+  - filter: {landuse: winter_sports}
+    min_zoom: *tier4_min_zoom
+    output: {kind: winter_sports, tier: 4}
+  # pier
+  - filter:
+      man_made: pier
+      way_area: {expr: "way_area > 0"}
+    min_zoom: *tier4_min_zoom
+    output: {kind: pier, tier: 4}
+  # wastewater_plant
+  - filter: {man_made: wastewater_plant}
+    min_zoom: *tier4_min_zoom
+    output: {kind: wastewater_plant, tier: 4}
+  # works
+  - filter: {man_made: works}
+    min_zoom: *tier4_min_zoom
+    output: {kind: works, tier: 4}
+  # bridge
+  - filter: {man_made: bridge}
+    min_zoom: *tier4_min_zoom
+    output: {kind: bridge, tier: 4}
+  # tower
+  - filter: {man_made: tower}
+    min_zoom: *tier4_min_zoom
+    output: {kind: tower, tier: 4}
+  # breakwater
+  - filter:
+      man_made: breakwater
+      way_area: {expr: "way_area > 0"}
+    min_zoom: *tier4_min_zoom
+    output: {kind: breakwater, tier: 4}
+  # water_works
+  - filter: {man_made: water_works}
+    min_zoom: *tier4_min_zoom
+    output: {kind: water_works, tier: 4}
+  # groyne
+  - filter:
+      man_made: groyne
+      way_area: {expr: "way_area > 0"}
+    min_zoom: *tier4_min_zoom
+    output: {kind: groyne, tier: 4}
+  # dike
+  - filter:
+      man_made: dike
+      way_area: {expr: "way_area > 0"}
+    min_zoom: *tier4_min_zoom
+    output: {kind: dike, tier: 4}
+  # cutline
+  - filter:
+      man_made: cutline
+      way_area: {expr: "way_area > 0"}
+    min_zoom: *tier4_min_zoom
+    output: {kind: cutline, tier: 4}
+
+  ############################################################
+  # TIER 5
+  ############################################################
+  # theme_park
+  - filter: {tourism: theme_park}
+    min_zoom: *tier5_min_zoom
+    output: {kind: theme_park, tier: 5}
+  # resort
+  - filter: {tourism: resort}
+    min_zoom: *tier5_min_zoom
+    output: {kind: resort, tier: 5}
+  # aquarium
+  - filter: {tourism: aquarium}
+    min_zoom: *tier5_min_zoom
+    output: {kind: aquarium, tier: 5}
+  # winery
+  - filter: {tourism: winery}
+    min_zoom: *tier5_min_zoom
+    output: {kind: winery, tier: 5}
+  # maze
+  - filter: {attraction: maze}
+    min_zoom: *tier5_min_zoom
+    output: {kind: maze, tier: 5}
+  # beach
+  - filter: {natural: beach}
+    min_zoom: *tier5_min_zoom
+    output: {kind: beach, tier: 5}
+  # glacier
+  - filter: {natural: glacier}
+    min_zoom: *tier5_min_zoom
```

Comment:
`glacier` wasn't in a tier in either Walkabout or Bubble Wrap, although labels are currently only showing up at zoom 10. I propose `tier3_min_zoom` for `glacier` landuse polygons instead.